OpenSim/Common/XsensDataReader.cpp

```diff
@@ -63,15 +62,14 @@ XsensDataReader::extendRead(const std::string& folderName) const {
                     dataRate = std::stod(tokens[3]);
                 }
             }
-            if (!labelsFound) { // Find indices for PacketCounter, Acc_{X,Y,Z}, Gyr_{X,Y,Z}, Mag_{X,Y,Z} on line 5
+            if (packetCounterIndex == -1) { // Find indices for PacketCounter, Acc_{X,Y,Z}, Gyr_{X,Y,Z}, Mag_{X,Y,Z} on line 5
```

Comment:
Can `packetCounterIndex == -1` ever be false given the `for` condition?

Reply:
No, good point 👍